str/tob-wgs/tob_eh_runner.py

```diff
@@ -49,6 +57,7 @@ def main(variant_catalog, sample_id_file):  # pylint: disable=missing-function-d
 
     ref_fasta = 'gs://cpg-common-main/references/hg38/v0/Homo_sapiens_assembly38.fasta'
     eh_regions = b.read_input(variant_catalog)
+    dependent_jobs = []
```

Comment:
I'd just rename that to `jobs`, as they're not all dependent, but that's a minor thing.